mollie/common_types.go

```diff
@@ -115,32 +115,147 @@ type PaginationLinks struct {
 	Documentation *URL `json:"documentation,omitempty"`
 }
 
-// CategoryCode specifies an industry or category.
-type CategoryCode uint
```

Comment:
Existing implementations of a type would be broken if we decided to remove it. I've always agreed that the API layer should be responsible for data validation and accuracy.

As a result, I recommend keeping the type but changing the type comment to state that it shouldn't be used any longer.

That way we create a new type without completely breaking existing implementations. WDYT?